src/database/utilities/createRecord.js

```diff
@@ -290,6 +306,9 @@ const createPatient = (database, patientDetails) => {
     nationality,
     ethnicity,
   });
+
+  nameNotes?.forEach(nameNote => createNameNote(database, nameNote));
```

Comment:
running this when selecting a remote patient, so running this to create all the name notes pulled